auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py

```diff
@@ -29,6 +29,20 @@ def get_gpu_available_memory():
 
 
 class Segresnet2dAlgo(BundleAlgo):
+    def pre_check_skip_algo(self):
+        if self.data_stats_files is None:
+            return
+        data_stats = ConfigParser(globals=False)
+        if os.path.exists(str(self.data_stats_files)):
+            data_stats.read_config(str(self.data_stats_files))
+        else:
+            data_stats.update(self.data_stats_files)
```

Comment:
may be check if self.data_stats_files  is a dict structure

Reply:
I copied how fill_template_config handles data_stats_files. Can be updated when fill_template_config is updated with this check.

Reply:
I copied how fill_template_config handles data_stats_files. Can be updated when fill_template_config is updated with this check.